app/lib/scorecard/models.dart

```diff
@@ -350,6 +352,53 @@ class ScoreCardData extends Object with FlagMixin {
     if (reportTypes == null || reportTypes.isEmpty) return false;
     return requiredTypes.every(reportTypes.contains);
   }
+
+  /// Returns [derivedTags] (if it is available) or calculates the tags based
+  /// on the [platformTags].
+  List<String> get currentTags {
+    if (derivedTags != null && derivedTags.isNotEmpty) return derivedTags;
+    if (platformTags == null || platformTags.isEmpty) return <String>[];
+    if (KnownPlatforms.all.every(platformTags.contains)) {
+      return <String>[
+        'sdk:${SdkTagValue.dart}',
+        'sdk:${SdkTagValue.flutter}',
+        'platform:${FlutterSdkRuntimeValue.android}',
+        'platform:${FlutterSdkRuntimeValue.ios}',
+        'platform:${FlutterSdkRuntimeValue.web}',
+        'runtime:${DartSdkRuntimeValue.native}',
+        'runtime:${DartSdkRuntimeValue.web}',
+      ];
+    } else if (platformTags.length == 1 &&
+        platformTags.single == KnownPlatforms.flutter) {
+      return <String>[
+        'sdk:${SdkTagValue.flutter}',
+      ];
+    } else if (platformTags.length == 1 &&
+        platformTags.single == KnownPlatforms.web) {
+      return <String>[
+        'sdk:${SdkTagValue.dart}',
+        'runtime:${DartSdkRuntimeValue.web}',
```

Comment:
This also implies `sdk:flutter` and `platform:web`.

Reply:
IIRC it is platformtags = ['web'] if there was use of `dart:html` or `dart:js`, that doesn't imply usability in flutter.